pgjdbc/src/main/java/com/yugabyte/ysql/LoadBalanceService.java

```diff
@@ -482,4 +525,8 @@ public String toString() {
       return "CloudPlacement: " + cloud + "." + region + "." + zone;
     }
   }
+
+  public enum LoadBalance {
```

Comment:
Shall we rename this enum to BALANCE_TYPE instead of LoadBalance? Ot will give a lot of clarity to the code reader.

Reply:
Renamed to `LoadBalanceType` as per the class name convention